packages/hardhat-ganache/src/ganache-service.ts

```diff
@@ -236,40 +285,86 @@ export class GanacheService {
         ? parseInt(url.port, 10)
         : DEFAULT_PORT;
 
-    const optionsToInclude = [
-      "accountsKeyPath",
-      "dbPath",
-      "defaultBalanceEther",
-      "totalAccounts",
-      "unlockedAccounts",
-    ];
-    for (const [key, value] of Object.entries(options)) {
-      if (value !== undefined && optionsToInclude.includes(key)) {
-        validatedOptions[this._snakeCase(key)] = value;
-        delete validatedOptions[key];
-      }
-    }
+    // This part was supposed to convert to snakecase which is not necessary anymore
+    // const optionsToInclude = [
+    //   "accountsKeyPath",
+    //   "dbPath",
+    //   "defaultBalanceEther",
+    //   "totalAccounts",
+    //   "unlockedAccounts",
+    // ];
+    // for (const [key, value] of Object.entries(options)) {
+    //   if (value !== undefined && optionsToInclude.includes(key)) {
+    //     // @ts-ignore
+    //     validatedOptions[this._snakeCase(key)] = value;
+    //     // @ts-ignore
+    //     delete validatedOptions[key];
+    //   }
+    // }
+
+    const resp = {
+      gasPrice: 20000000000,
+      gasLimit: 6721975,
+      wallet: {
+        totalAccounts: validatedOptions.totalAccounts,
+        defaultBalance: validatedOptions.defaultBalanceEther,
+        // hdPath:validatedOptions.hdPath,
+        // unlockedAccounts:validatedOptions.unlockedAccounts,
+        // accountKeysPath:validatedOptions.accountKeysPath,
+        mnemonic: validatedOptions.mnemonic,
+      },
+    };
 
-    return validatedOptions;
-  }
+    return resp;
 
-  private _registerSystemErrorsHandlers() {
-    const server = this._server;
-
-    // Add listener for general server errors
-    server.on("error", function (err: any) {
-      if (
-        GanacheService.error === undefined &&
-        err !== undefined &&
-        err !== null
-      ) {
-        log("An error occurred in GanacheService\n", err);
-        GanacheService.error = err;
-      }
-    });
+    // TODO: make a function that correctly does this conversion:
+    return {
```

Comment:
@fvictorio Adding all the custom options is the main task that is left to do but the rest works. Please take a first pass a reviewing my work when you get the chance

Reply:
Hey @joelamouche, I'll be a bit busy the next couple of days, but I'll try to take a look next week.